NebulaWorld/Chat/Commands/PlayerDataCommandHandler.cs

```diff
@@ -0,0 +1,167 @@
+﻿#region
+
+using NebulaWorld.MonoBehaviours.Local.Chat;
+using System.Collections.Generic;
+using NebulaAPI.GameState;
+using HarmonyLib;
+using NebulaModel.DataStructures.Chat;
+using System.IO;
+using NebulaModel;
+using NebulaModel.Logger;
+using NebulaAPI.DataStructures;
+using NebulaModel.Networking;
+using NebulaModel.Packets.Players;
+
+#endregion
+
+namespace NebulaWorld.Chat.Commands;
+
+public class PlayerDataCommandHandler : IChatCommandHandler
+{
+    public void Execute(ChatWindow window, string[] parameters)
+    {
+        if (parameters.Length < 1)
+        {
+            throw new ChatCommandUsageException("Not enough arguments!".Translate());
+        }
+
+        // Due to dependency order, get SaveManager.playerSaves by reflection
```

Comment:
This is fine for now but I was actually just thinking that maybe it would make more sense if SaveManager was in NebulaWorld instead of NebulaNetwork. I guess that would solve this issue too without needing reflection, though I'm not sure if there'd be any issues with that.

Reply:
If SaveManager is in NebulaWorld, then `SaveManager.LoadServerData` will not able to be called in `Server.Start()`. It will have to hook in other place.
Hmm.. I guess this is fine and can further decouple the part in Server?

Reply:
That would still work anyway since NebulaNetwork references NebulaWorld, no?

Reply:
oh, my bad. I've mistaken the order